department-of-veterans-affairs/va.gov-cms
# VACMS-2693 2694 2695 - service locations interactions

## Description
See #2693 #2694 #2695 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/90660814-6af6bd80-e214-11ea-863b-eb4bb7bd0283.png)
![image](https://user-images.githubusercontent.com/2404547/90660841-6fbb7180-e214-11ea-8e2e-5569fda80601.png)

## QA steps

- As an admin, go to `node/add/health_care_local_health_service`
- Add a second service_location paragraph
- Toggle the first and second instances of `Use facility address?` and confirm they show and hide the corresponding address fieldset appropriately.
- Visually verify that the `Phone` fieldset paragraph in service_locations paragraph is collapsed
- Visually verify that `Use main facility phone number?` field in the phone fieldset in service_location paragraphs is not required.
